includes/classes/db/mysql/define_queries.php

```diff
@@ -29,6 +29,5 @@
                      AND opb.orders_id = o.orders_id
                      AND p.products_status = 1
                      GROUP BY p.products_id, p.products_image
-                     ORDER BY date_purchased desc, p.products_id
-                     LIMIT %u");

```

Comment:
Why did the `LIMIT %u` get removed?  It seems that now **_all_** also-purchases products are going to be pulled with this query.

Reply:
All are pulled and then MAX_DISPLAY_ALSO_PURCHASED are selected from the set.  See `includes/modules/also_purchased_products.php` line 15.

Reply:
A better way to describe the change is that the LIMIT has been relocated from the DEFINE SQL to a parameter on the `$db->ExecuteRandomMulti` function call.

Technically the result is the same.